app/views/shared/admin_settings/_site_settings.html.erb

```diff
@@ -28,6 +28,19 @@
       </div>
     </div>
   </div>
+  <div class="row">
+    <div class="col-12">
+      <div class="mb-7 form-group">
+        <label class="form-label"><%= t("administrator.site_settings.authentication.title") %></label>
+        <label class="form-label text-muted"><%= t("administrator.site_settings.authentication.info") %></label>
+        <div class="row gutters-xs">
```

Comment:
You should use the tabler checkbox instead of just the standard checkbox. You can see how here: https://preview.tabler.io/form-elements.html